torch/_torch_docs.py

```diff
@@ -3,6 +3,47 @@
 import torch._C
 from torch._C import _add_docstr as add_docstr
 
+
+def parse_kwargs(desc):
+    """Maps a description of args to a dictionary of {argname: description}.
+    Input:
+        out (Tensor): the output tensor
```

Comment:
Cool. Not a blocker, but it would be nice if the input can be formatted as string literals... It took me a while to understand why out is an input lol.